theano/tensor/nnet/tests/test_abstract_conv.py

```diff
@@ -296,7 +296,7 @@ def test_all(self):
                     for b in self.border_modes:
                         try:
                             self.tcase(i, f, s, db, dflip,
-                                       dprovide_shape, fd)
+                                       dprovide_shape)
```

Comment:
This is the wrong fix. It will skip all filters_dilation tests. Maybe we need to find the implementation of tcase that don't support this case and extend it.